web/modules/custom/employees/employees.module

```diff
@@ -67,6 +67,15 @@ function employees_user_format_name_alter(&$name, $account)
   }
 }
 
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function employees_form_openid_connect_login_form_alter(&$form, &$form_state)
```

Comment:
@ccdatapdx , this would have broken our login had I merged. All functions in the employees module need to be prefixed with `employees_`. If you get another story like this one, keep in mind that you can't just copy portlandor functions without a little editing 😄